nessie/jobs/scheduling.py

```diff
@@ -189,11 +189,13 @@ def run_startup_jobs(_app):
     from nessie.jobs.create_asc_schema import CreateAscSchema
     from nessie.jobs.create_metadata_schema import CreateMetadataSchema
     from nessie.jobs.create_rds_indexes import CreateRdsIndexes
+    from nessie.jobs.create_sis_advising_notes_schema import CreateSisAdvisingNotesSchema
     from nessie.jobs.create_student_schema import CreateStudentSchema
 
     if _app.config['JOB_SCHEDULING_ENABLED'] and os.environ.get('NESSIE_ENV') != 'test':
         _app.logger.info('Checking for required schemas...')
         CreateAscSchema().run()
         CreateMetadataSchema().run()
         CreateRdsIndexes().run()
+        CreateSisAdvisingNotesSchema().run()
```

Comment:
It seems premature to put this into the scheduler. OK if we leave it "manual-only" for a while?